src/commands/runDraftTool/runBuildOnAcr.ts

```diff
@@ -0,0 +1,77 @@
+import {KnownRunStatus} from '@azure/arm-containerregistry';
+import {buildAcrImage} from '../../utils/dockerExtension';
+import {longRunning} from '../../utils/host';
+import {State, StateApi} from '../../utils/state';
+import {Context} from './model/context';
+import {parseAzureResourceId} from '@microsoft/vscode-azext-azureutils';
+import * as vscode from 'vscode';
+import {image} from '../../utils/acr';
+import {CompletedSteps} from './model/guidedExperience';
+
+export async function runBuildAcrImage(
+   {actionContext, extensionContext}: Context,
+   completedSteps: CompletedSteps
+) {
+   const state: StateApi = State.construct(extensionContext);
```

Comment:
Does this still work across VSCode restarts?

Reply:
Yes it does.